### PR TITLE
WIP:model: Adding Vowpal wabbit models

### PR DESCRIPTION
**Pull request by 0dust**
Issue #328

### CODE REVIEW

**Comment by 0dust** [comment]
@rahulunair This is the basic skeleton that currently can't process text. Please have a look.

File: model/vowpalWabbit/dffml_model_vowpalWabbit/vw_base.py [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+            if arg in self.vwcmd:
+                raise InputError(
+                    f"Passing data through {arg} option is not supported. Pass the input data using dffml sources."
+                )
+            # if not vwcmd[0].startswith('-'):
+            #     raise InputError(f"Passing filename in vwcmd is not supported. Pass the input data using dffml sources.")
+        # TODO handle --save_resume
+        if "--examples" in self.vwcmd:
+            raise InputError(
+                "Currently dffml sources do not support reading specified number of rows."
```
Comment (pdxjohnny) [inline model/vowpalWabbit/dffml_model_vowpalWabbit/vw_base.py]: Added https://github.com/intel/dffml/issues/380 to track

File: model/vowpalWabbit/tests/test_vw_integration.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                """
+            )
+            + "\n"
+        )
+        # Create the prediction data
+        predict_filename = self.mktempfile() + ".csv"
+        pathlib.Path(predict_filename).write_text(
+            inspect.cleandoc(
+                """
+                Clump_Thickness,Uniformity_of_Cell_Size,Uniformity_of_Cell_Shape,Marginal_Adhesion,Single_Epithelial_Cell_Size,Bare_Nuclei,Bland_Chromatin,Normal_Nucleoli,Mitoses,Class
```
Comment (pdxjohnny) [inline model/vowpalWabbit/tests/test_vw_integration.py]: We need to make sure this is randomly generated

**Comment by pdxjohnny** [comment]
- Issues with dependencies
- Pip install from PyPi didn’t work
- Had to install from source
- For CI, need to add bullet 2 under Linux of above install guide to the .github/workflows/testing.yml in https://github.com/intel/dffml/blob/8a223a2d327be05ffa980d3b0ff5db298aabf191/.github/workflows/testing.yml#L49-L63

**Comment by 0dust** [comment]
@rahulunair Please have a look.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Added
+- Added Vowel Wabbit models
```
Comment (pdxjohnny) [inline CHANGELOG.md]: Let's remove the `Added` (since it's redundant, Added, Added ... models) Sorry I still haven't added this to the docs #81

**Pull request closed without merging**